modules/platforms/cpp/tests/client-test/main.cpp

```diff
@@ -74,21 +74,21 @@ void before_all() {
 
 int main(int argc, char **argv) {
     int res = 0;
-    before_all();
+//    before_all();
```

Comment:
Is this a temporary change?

Reply:
Damn, yeah, it was for the debugging. I'll reverse it.